LICENSE.md

```diff
@@ -0,0 +1,27 @@
+# WorkExp-CompWorkshop
+
+This work is intended for educational purposes.
+
+Copyright 2019 Science and Technology Facilities Council, U.K. Research and Innovation
+
+```
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+```
+
+# Invent Your Own Computer Games with Python 3rd Edition
+
+This work is based on the book “Invent Your Own Computer Games with Python 3rd Edition” by Al Sweigart (ISBN-10 0982106017), and is licensed under Creative Commons Attribution-Noncommercial-Share Alike 3.0 United States License.
```

Comment:
```suggestion
This work is based on the book “Invent Your Own Computer Games with Python 3rd Edition” by Al Sweigart (ISBN-10 0982106017) which is licensed under a Creative Commons Attribution-Noncommercial-Share Alike 3.0 United States License.
```

It was ambiguous as to whether you were referring to this work or the book.

Reply:
Agreed, this is a better structure.